Source/WebService/Blackjack.WebService/ClientApp/src/app/types/player.ts

```diff
@@ -6,10 +6,15 @@ export class Player extends Gamer {
    */
   public name: string;
 
+  public updateHand(cards: string[]): void {
+    super.updateHand(cards);
+    this.getStatus();
+  }
+
   /**
    * Displays the name, shown cards, and score of the player
    */
-  public getStatus(): string {
-    return super.printStatus(this.name);
+  private getStatus(): void {
```

Comment:
This method can be removed and the super can just be called directly if you have nothing to add in addition to the base method being invoked

Reply:
I get what you're saying but referencing your comment above asking about the updatehand(0) line, if I make the method public, should I keep this method strictly so the only possible string going to the super method is the players name? Or am I missing an obvious solution to that

Reply:
I get what you're saying but referencing your comment above asking about the updatehand(0) line, if I make the method public, should I keep this method strictly so the only possible string going to the super method is the players name? Or am I missing an obvious solution to that

Reply:
In this specific comment I'm not advocating this private method be made public. 

I'm saying it should be removed entirely and anywhere using it inside of player.ts should call the super variant directly

This is because your private player.ts variant is not doing anything extra before calling the super variant

So you're adding an extra step here for no reason.

This may be a moot point based on feedback given elsewhere as I think some other recommendations will make this method necessary again.